ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/JsonResponseStreamer.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright ConsenSys AG.
```

Comment:
New files should be Copyright Hyperledger Besu Contributors

Reply:
you are right, too fast cut and paste :)